app/code/Magento/Eav/Model/Entity/Attribute/Frontend/AbstractFrontend.php

```diff
@@ -49,22 +49,22 @@
     /**
      * @param \Magento\Eav\Model\Entity\Attribute\Source\BooleanFactory $attrBooleanFactory
      * @param CacheInterface $cache
-     * @param StoreResolverInterface $storeResolver
+     * @param StoreManagerInterface $storeManager
      * @param array $cacheTags
      * @codeCoverageIgnore
      */
     public function __construct(
         \Magento\Eav\Model\Entity\Attribute\Source\BooleanFactory $attrBooleanFactory,
         CacheInterface $cache = null,
-        StoreResolverInterface $storeResolver = null,
+        StoreManagerInterface $storeManager = null,
```

Comment:
Really? It only existed in Magento 2.1.6, in no other version. It would very much surprise me if any 3rd party already extends this class.

But if you want me to add it as a new param, then I can do that.
Please let me know.